package.json

```diff
@@ -17,6 +17,7 @@
     "build:preview": "npm run build:webpack && npm run build:hugo:preview",
     "build:hugo": "hugo -d ../dist -s site -v",
     "build:hugo:preview": "npm run build:hugo -- -D -F",
+    "build:hugo:debug": "hugo -d ../dist -s site -v --templateMetrics --debug",
```

Comment:
Works great!

Reply:
I figured it's handy to be able to run this easily :)